### PR TITLE
Set up boilerplate

### PR DESCRIPTION
**Pull request by yoko-8**
Review boilerplate

### CODE REVIEW

**Review by beanjermin — APPROVED** [review]
Looks good!

**Review by seguyy — COMMENTED** [review]
Test

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (seguyy) [inline client/src/components/App.jsx:1]: Looks good!

**Review by seguyy — COMMENTED** [review]
Test

**Comment by davidguy3237** [comment]
[Set up boilerplate/initialize repository](https://trello.com/c/V6CdLIgp/8-set-up-boilerplate-initialize-repository)

**Pull request merged**